hoot-core/src/main/cpp/hoot/core/io/ApiDbReader.cpp

```diff
@@ -201,8 +201,7 @@ void ApiDbReader::_updateMetadataOnElement(ElementPtr element)
     if (r)
     {
       r->setType(tags["type"]);
-      //I don't think OSM non-hoot metadata tags should be removed here...
-      //tags.remove("type");
+      tags.remove("type");
```

Comment:
If the `type` tag isn't removed here, it shows up twice in the data.  Duplicate tags are annoying.

Reply:
Might as well remove the comment since you've proven the tag can be removed there.